docs/infra/self_hosted_github_runners.md

```diff
@@ -0,0 +1,116 @@
+# Self Hosted GitHub Runners
+
+A self-hosted runner is a system that you deploy and manage to execute jobs from GitHub Actions on GitHub.com.
+This can be useful for
+
+* running automation scripts
+* deploying production / pre-production servers
+
+To use self-hosted runners, you will need to be an administrator of the GitHub project.
+If you can access `/settings/actions/runners` then you have the right access permissions.
+
+![GitHub Settings Runners](/docs/assets/github/github_settings_runners.png)
+
+## Create Self-Hosted Runner
+
+You can get instructions on how to add a self-hosted actions runner from the settings page (change the organization/project information in the
```

Comment:
Unnecessary carriage return after "the".

While not as obvious, there are unnecessary carriage returns in several places below as well, including lines 25, 26, 31, 45, 46, 48, 96